### PR TITLE
chore(TS): polish files

### PR DESCRIPTION
**Pull request by ShaMan123**
## Description

Mainly remove junk jsdoc (`memberof fabric.` that is dead), forgotten imports, types, chainable return values of StaticCanvas, etc.
I commented the interesting stuff.

### CODE REVIEW

File: src/shapes/object.class.ts [hunk 1]
```diff
@@ -928,7 +934,7 @@ export class FabricObject<
    * @return {String}
    */
   toString() {
-    return '#<fabric.' + capitalize(this.type) + '>';
+    return `#<${capitalize(this.type)}>`;
```
Comment (ShaMan123) [inline src/shapes/object.class.ts:937]: minor

File: src/shapes/object.class.ts [hunk 2]
```diff
@@ -1679,10 +1680,11 @@ export class FabricObject<
    * @param {Boolean} [options.enableRetinaScaling] Enable retina scaling for clone image. Introduce in 1.6.4
    * @param {Boolean} [options.withoutTransform] Remove current object transform ( no scale , no angle, no flip, no skew ). Introduced in 2.3.4
    * @param {Boolean} [options.withoutShadow] Remove current object shadow. Introduced in 2.4.2
-   * @return {fabric.Image} Object cloned as image.
+   * @return {Image} Object cloned as image.
    */
   cloneAsImage(options: any) {
     const canvasEl = this.toCanvasElement(options);
+    // TODO: how to import Image w/o an import cycle?
     return new fabric.Image(canvasEl);
```
Comment (ShaMan123) [inline src/shapes/object.class.ts:1688]: TODO: how to import Image w/o an import cycle?

THoguhts?

**Review by ShaMan123 — COMMENTED** [review]
Mostly removed JSDOC
`memberof fabric.util`

**Review by asturur — APPROVED** [review]

**Comment by ShaMan123** [comment]
Yes!
Great

**Pull request merged**